web/ajax/generate.php

```diff
@@ -12,7 +12,11 @@
   die('Request parameters missing');
 }
 
+require_once( '../includes/config.php' );
```

Comment:
This shouldn't be necessary as it is included in index.php.  Doesn't hurt to have it though.